### DISCUSSION TITLE
Error 400 - Untrusted host with TRUSTED_HOSTS

### DISCUSSION THREAD

**Discussion by cdebackere** [kimai/kimai#3568] [On-Premise: Installation & Updates] [answered]
I made a docker compose install.
All work fine locally by going to the LAN_IP:8008 on the LAN

However, we also configured a nginx reverse proxy (in another docker container) that handles all incomming http/https calls on a named basis. For the kimai service this is lets say kimai.mydom.com
The forwarding and certificate works fine when hhtp(s)-ing to kimai.mydom.com
I get redirected correctly http->https
I then get redirected to kimai.mydom.com/nl/homepage

And then I get an Error 400 in the browser. In the logs I see:
`NOTICE: PHP message: [2022-10-04 14:23:47] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\BadRequestHttpException: "Untrusted Host "_kimai.mydom.com_"." at /opt/kimai/vendor/symfony/http-kernel/HttpKernel.php line 84 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\BadRequestHttpException(code: 0): Untrusted Host \"_kimai.mydom.com_\". at /opt/kimai/vendor/symfony/http-kernel/HttpKernel.php:84, Symfony\\Component\\HttpFoundation\\Exception\\SuspiciousOperationException(code: 0): Untrusted Host \"_kimai.mydom.com_\". at /opt/kimai/vendor/symfony/http-foundation/Request.php:1197)"} []`

In the kimai docker compose config file I added _kimai.mydom.com_ to the TRUSTED_HOSTS of the nginx . I restarted the compose several times.

I also tried to copy that info in a TRUSTED_PROXIES, to no avail.

Any hint?

THx,
C.

**Comment by kevinpapst** [comment, marked as answer]
Wrong repository, please go here and read the docs: https://tobybatch.github.io/kimai2/

Here is help as well: https://symfony.com/doc/current/deployment/proxies.html

Afaik you must add the IP of your reverse proxy, not sure though...
Last option is to ask here: https://github.com/tobybatch/kimai2

**Reply by cdebackere** [reply]
thx, and my apologies for the mistake
C.
